node:events:353
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::3000
    at Server.setupListenHandle [as _listen2] (node:net:1295:16)
    at listenInCluster (node:net:1343:12)
    at Server.listen (node:net:1429:7)
    at Application.listen (/Users/Diatta/WebstormProjects/seaweedfs-node-daemon/node_modules/koa/lib/application.js:82:19)
    at /Users/Diatta/WebstormProjects/seaweedfs-node-daemon/dist/index.bundle.js:342:22
Emitted 'error' event on Server instance at:
    at emitErrorNT (node:net:1322:8)
    at processTicksAndRejections (node:internal/process/task_queues:80:21) {
  code: 'EADDRINUSE',
  errno: -48,
  syscall: 'listen',
  address: '::',
  port: 3000
}
node:events:353
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::3000
    at Server.setupListenHandle [as _listen2] (node:net:1295:16)
    at listenInCluster (node:net:1343:12)
    at Server.listen (node:net:1429:7)
    at Application.listen (/Users/Diatta/WebstormProjects/seaweedfs-node-daemon/node_modules/koa/lib/application.js:82:19)
    at Object.<anonymous> (/Users/Diatta/WebstormProjects/seaweedfs-node-daemon/dist/index.bundle.js:331:22)
    at Module._compile (node:internal/modules/cjs/loader:1108:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10)
    at Module.load (node:internal/modules/cjs/loader:973:32)
    at Function.Module._load (node:internal/modules/cjs/loader:813:14)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:12)
    at node:internal/main/run_main_module:17:47
Emitted 'error' event on Server instance at:
    at emitErrorNT (node:net:1322:8)
    at processTicksAndRejections (node:internal/process/task_queues:80:21) {
  code: 'EADDRINUSE',
  errno: -48,
  syscall: 'listen',
  address: '::',
  port: 3000
}
node:events:353
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::3000
    at Server.setupListenHandle [as _listen2] (node:net:1295:16)
    at listenInCluster (node:net:1343:12)
    at Server.listen (node:net:1429:7)
    at Application.listen (/Users/Diatta/WebstormProjects/seaweedfs-node-daemon/node_modules/koa/lib/application.js:82:19)
    at Object.<anonymous> (/Users/Diatta/WebstormProjects/seaweedfs-node-daemon/dist/index.bundle.js:331:22)
    at Module._compile (node:internal/modules/cjs/loader:1108:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10)
    at Module.load (node:internal/modules/cjs/loader:973:32)
    at Function.Module._load (node:internal/modules/cjs/loader:813:14)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:12)
    at node:internal/main/run_main_module:17:47
Emitted 'error' event on Server instance at:
    at emitErrorNT (node:net:1322:8)
    at processTicksAndRejections (node:internal/process/task_queues:80:21) {
  code: 'EADDRINUSE',
  errno: -48,
  syscall: 'listen',
  address: '::',
  port: 3000
}
